ops/terraform/services/server/server-regression/iam.tf

```
@@ -326,3 +326,86 @@ resource "aws_iam_role" "glue_trigger" {
     aws_iam_policy.glue.arn
   ]
 }
+
+resource "aws_iam_role" "spice_refresh_role" {
+  name = "bfd-${local.env}-${local.service}-spice-trigger-role"
```

```suggestion
  name = local.spice_trigger_lambda_name
```

nit: _Definitely_ not a blocker to approval, but the usual convention is to name Lambda IAM Roles the same name as the Lambda itself. So, this could just be `local.spice_trigger_lambda_name`.